src/routes.json

```diff
@@ -323,5 +323,11 @@
         "pattern": "^/go/?(\\?.*)?$",
         "routeAlias": "/go/?\\??",
         "redirect": "/tips"
+    },
```

Comment:
You also need to add the actual `/research` route that points at your new view.

Reply:
Hm? I'm not really sure what you mean here.

Reply:
You need a route for `/research` that actually specifies your view. You can looks at the `/developers` route for an example (https://github.com/LLK/scratch-www/blob/380be6b5c1dc537b787bef34005b3610e4b04fd4/src/routes.json#L89). Also note that we're trying to keep the routes alphabetized, followed by all the redirects (also alphabetized).